src/trivia/config.h.cmake

```diff
@@ -61,6 +61,13 @@
  * showing fiber call stack.
  */
 #cmakedefine ENABLE_BACKTRACE 1
+
+/*
+ * Defined if configured with ENABLE_BACKTRACE on AARCH64 and
+ * '-mbranch-protection' compile flag is passed.
+ */
+#cmakedefine ENABLED_BRANCH_PROTECTION
```

Comment:
Nit: should be

```
#cmakedefine ENABLED_BRANCH_PROTECTION 1
```

?

Reply:
I prefer using `#ifdef`, when possible, rather than `#if`: according to CMake [docs](https://cmake.org/cmake/help/latest/command/configure_file.html) this will be replaced with 
```C
#define ENABLED_BRANCH_PROTECTION
```
or
```C
/* #undef ENABLED_BRANCH_PROTECTION */
```

Reply:
I also decided to change the prefix to `WITH_` — sounds more suitable.